cmd/install/install.go

```diff
@@ -74,6 +74,16 @@ func installers() (i []installer) {
 	return
 }
 
+func zshrcPath() string {
```

Comment:
Can you please add documentation?
What does this function look for? in what cases?

Reply:
Added documentation